src/pages/index.js

```diff
@@ -18,80 +19,202 @@ import {
   profileEditForm,
   profileTitleInput,
   profileDescriptionInput,
+  modalButton,
+  formValidatorConfig,
+  profileAvatarButton,
+  editAvatarForm,
+  // profileAvatar,
 } from "../utils/constants.js";
 import "./index.css";
 
 const userinfoComponent = new Userinfo(
   selectors.profileTitle,
-  selectors.profileDescription
+  selectors.profileDescription,
+  selectors.profileAvatar
 );
 
 //Api
-const api = new Api({
```

Comment:
When I update the user info it doesn't change on the server. When I reload the page the initial info is shown.